Makefile

```diff
@@ -29,6 +29,10 @@ build:  ## Build binary
 	go generate ./...
 	go build -v -ldflags="-w -s -X github.com/0xERR0R/blocky/util.Version=${VERSION} -X github.com/0xERR0R/blocky/util.BuildTime=${BUILD_TIME}" -o $(BIN_OUT_DIR)/$(BINARY_NAME)$(BINARY_SUFFIX)
 
+build-static:  ## Build static binary
+	go generate ./...
+	go build -tags static -v -ldflags="-linkmode external -extldflags -static -X github.com/0xERR0R/blocky/util.Version=${VERSION} -X github.com/0xERR0R/blocky/util.BuildTime=${BUILD_TIME}" -o $(BIN_OUT_DIR)/$(BINARY_NAME)$(BINARY_SUFFIX)
```

Comment:
Never worked with static binaries. I assume, the binary is bigger (file size), but includes everything to be really portable, correct? This must be also changed in https://github.com/0xERR0R/blocky/blob/8180858efc7dc39b760806b23d847ad01f3d4d81/.goreleaser.yml#L27

Reply:
"the binary is bigger (file size), but includes everything to be really portable" - correct.

Its not neccesarry to build statically outside of a scratch environment as most setups will already include things like tzdata and rootcerts. Most likely this will only waste spache for >90% of the users using binaries outside of containers. 😅